### PR TITLE
Update typings of Account capabilities

### PR DESCRIPTION
**Pull request by abalarin**
## Description

Attempt at refactoring Account capabilities.

## Type of Change
- Non breaking change ('update', 'change')

### CODE REVIEW

File: packages/manager/src/App.tsx [hunk 1]
```diff
@@ -307,7 +306,7 @@ interface StateProps {
   username: string;
   documentation: Linode.Doc[];
   isLoggedInAsCustomer: boolean;
-  accountCapabilities: AccountCapability[];
+  accountCapabilities: Account['capabilities'][];
```
Comment (martinmckenna) [inline packages/manager/src/App.tsx]: This is incorrectly typed now.

`Account['capabilities']` is an array of `AccountCapabilities` but you've got this typed as a 2D array. I think most of the files could have been unchanged and just left as `AccountCapability[]`.

**Comment by martinmckenna** [comment]
@abalarin any update on this?

**Comment by abalarin** [comment]
@martinmckenna unfortunately no 😭 
I was hoping to work on this on TDT but have not had a chance... I'm happy to close this as it may be some time before I get a chance to take a look at this again, but if no one gets around to it I can keep working on it.

**Comment by martinmckenna** [comment]
kk. In that case, we'll close for now

**Pull request closed without merging**